src/ components/TableBody/TableRow/index.js

```diff
@@ -0,0 +1,46 @@
+import React from "react"
+import "./TableRow.scss"
+import SlidingSubmenu from "../SlidingSubmenu/"
+import { CSSTransitionGroup } from "react-transition-group"
+import iconAuthor from "../../../images/kara.png"
+const TableRow = ({ arrData, isMenuHidden }) => {
+  return (
+    <div className="tb-body-wrapper">
+      <div
+        className={
+          isMenuHidden === true
```

Comment:
If it's boolean you could just:
```suggestion
          isMenuHidden
```